### PR TITLE
Updated our definition of 'compliant' and how it is achieved in CFI

### PR DESCRIPTION
**Pull request by eddie-knight**
Added a new section in the primary readme to answer this commonly asked question

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,60 @@ Cloud services controls and tests are used to demonstrate adherence with regulat
...
+As detailed in [the following section](#services--workflows), our community focuses on these three key offerings.
+
+Our compliance definition comes from the an extension of a baseline into a
+comprehensive standard that is suited to accelerate adoption in highly regulated environements.
+
+### Defining Compliant Infrastructure
+
+It must be noted that every institution is made up of highly skilled professionals who collaborate to define what _compliance_ means within their firm.
+As such, _compliance_ may mean something different from one institution to the next. The goal of CFI is _not_ to create a single solution that all firms must adhere to,
+instead our goal is to streamline adoption and free up security teams to focus on less redundant activities.
```
Comment (thinkl33t) [inline README.md:28]: `s/less/fewer/` i think would make this sentence less ambiguous.

**Comment by thinkl33t** [comment]
All reads well and matches with my understanding of what we're trying to achieve.  There is one sentence which was a little ambiguous, i've chucked a suggestion onto it.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,60 @@ Cloud services controls and tests are used to demonstrate adherence with regulat
...
+A template for the Service Approval Accelerator can be found [here](templates/ServiceApprovalAcceleratorTemplate.md).
+
+### Creating Compliant Infrastructure
+
+Infrastructure as Code (IaC) is a key component of every modern firm's infrastructure efforts. 
+
+Many times, however, our community has seen the task of IaC development fall on the shoulders of engineers and developers who are not fully familiar with the required technologies.
+This results in added time, stress, and risks as teams become familiar with the necessary technologies.
+
+In other cases, we see nearly identical IaC efforts being undertaken by multiple teams within a single business unit, resulting in uncessesary costs and delays.
```
Comment (thinkl33t) [inline README.md:41]: s/uncessesary/unnecessary/

File: README.md [hunk 3]
```diff
@@ -11,11 +11,60 @@ Cloud services controls and tests are used to demonstrate adherence with regulat
...
+individually reviewed by your security teams and imported to a firm registry for extention or consumption.
+
+### Validating Compliant Infrastructure
+
+A key element to compliance is _validating the result of an activity_. This is a pillar to the CFI efforts.
+
+In addition to technology-specific automated tests and scans in the code repository, our community believes that it is essential to provide
+the tools necessary to ensure that infrastructure is compliant after it has been deployed into an environment.
+
+In some cases, full compliance may only be possible (or preferrable) when deployed behind a firewall or with other similar contexts not reflected by the IaC code.
```
Comment (thinkl33t) [inline README.md:58]: preferable

Comment (eddie-knight) [reply README.md]: corrected

**Pull request merged**